README.md

```diff
@@ -2,16 +2,43 @@
 
 Source plugin for pulling documents into Gatsby from a Strapi API.
 
-> ⚠️ This version of `gatsby-source-strapi` is only compatible with Strapi v3 at the moment. We are currently working on a v4 compatible version.
+> ⚠️ This version of `gatsby-source-strapi` is only compatible with Strapi v4.
+
+<details>
+<summary><strong>Table of contents</strong></summary>
+
+- [gatsby-source-strapi](#gatsby-source-strapi)
+  - [Install](#installing-the-plugin)
+    - [Basic usage](#basic-usage)
+    - [Advanced usage](#advanced-usage)
+      - [Deep queries populate](#deep-queries-populate)
+      - [Draft content](#draft-content)
+      - [Image optimisation](#image-optimisation)
+      - [Rich text field](#rich-text-field)
+      - [Components](#components)
+      - [Dynamic zones](#dynamic-zones)
+  - [Restrictions and limitations](#restrictions-and-limitations)
+
+</details>
 
 ## Installing the plugin
 
-```sh
+```shell
```

Comment:
I'd recommend splitting yarn and npm into two different code blocks, with "using yarn" and "using npm" outside of the code blocks. This way users can use Github's copy snippet feature and only get what they need